libs/sdk-backend-spi/src/workspace/ldm/datasets.ts

```diff
@@ -9,9 +10,16 @@ import { IDataset } from "../fromModel/ldm/datasets";
  */
 export interface IWorkspaceDatasetsService {
     /**
-     * Receive all workspace datasets
+     * Receive all workspace csv datasets
      *
-     * @returns promise of workspace datasets
+     * @returns promise of workspace csv datasets
      */
     getDatasets(): Promise<IDataset[]>;
+
+    /**
+     * Receive all workspace all datasets
+     *
+     * @returns promise of workspace datasets
+     */
+    getAllDatasets(): Promise<IMetadataObject[]>;
```

Comment:
I suggest to rename to getAllDatasetsMeta() to indicate that the function returns just the metadata and not full information about the data set.

Reply:
updated